src/main/java/dev/jbang/net/TrustedSources.java

```diff
@@ -85,7 +85,7 @@ public boolean isURLTrusted(URI url) throws URISyntaxException {
 			}
 
 			URI parsedTrustedSource;
-			if (trustedSource.startsWith("https://")) {
+			if (trustedSource.startsWith("http://")) {
```

Comment:
Now it won't work with https so that does not work as fix.

Reply:
Okay I can just add a or condition in here then @maxandersen what do you suggest.

Reply:
Yes. That seems reasonable. 

I highly expect the tests to fail with the current suggestion so I suggest you run the tests, then see what fails and add a test that test for http based URL. Then fix it so all the tests passes.

Reply:
@maxandersen we have this method already present. 
![image](https://github.com/jbangdev/jbang/assets/42388388/8c76092f-a9d0-4872-9921-ace97da0b0d4)

Reply:
Yes. So add a test that uses http so we can verify it failed before and fixed now with the change.